src/Hubbard/hubbard_occupancies_derivatives.hpp

```diff
@@ -5,9 +5,9 @@
 
 // gradient of beta projectors. Needed for the computations of the forces
 
-void Hubbard_potential::compute_occupancies_derivatives(K_point& kp,
-                                                        Q_operator<double_complex>& q_op, // overlap operator
-                                                        mdarray<double_complex, 6>& dn_)                // Atom we shift
+void Hubbard::compute_occupancies_derivatives(K_point& kp,
+                                              Q_operator<double_complex>& q_op, // overlap operator
+                                              mdarray<double_complex, 6>& dn_)                // Atom we shift
```

Comment:
function arguments have two trailing underscores (dn__ in this example), class members one underscore